docs/user_guide.rst

```diff
@@ -61,6 +62,50 @@ This sections describes storage backend operation supported by InfraSIM.
 * Drive erasure.
 
   Drive erasure feature is implemented in Qemu code. After erasing, all data residing on a disk drive will be overwritten with all zero. Below are examples of SAS and SATA drive erasure performed in Ubuntu 16.04.
+    
```

Comment:
Sounds make sense. How about to merge this PR if no more question for page file related content? @fub2 
Since PR for drive erase is already merged, we plan to revise for your comments after Julie's PR for time customizing merged.